### PR TITLE
slightly expand discussion of delegated credentials and what client needs for support (by summarizing RFC9345)

### PR DESCRIPTION
**Pull request by danwing**

### CODE REVIEW

File: draft-rbw-add-encrypted-dns-forwarders.md [hunk 1]
```diff
@@ -357,16 +357,18 @@ making it impossible to identify whether the delegated credential is
 issued to the intended CPE or an "evil-twin" CPE. This drawback can be corrected
 by enhancing {{?RFC9345}} to include a string that uniquely identifies the
 delegated credential (e.g., including hash of customer id or other unique identifier
-in the FQDN to result in "HASH.cpe.example.com").
-
+in the FQDN such as "HASH.cpe.example.com").  This document assumes such an
```
Comment (boucadair) [inline draft-rbw-add-encrypted-dns-forwarders.md]:
```suggestion
in the FQDN such as "HASH.cpe.example.com").

  > For the sake of simplifying the analysis, this document assumes such an
```

File: draft-rbw-add-encrypted-dns-forwarders.md [hunk 2]
```diff
@@ -357,16 +357,18 @@ making it impossible to identify whether the delegated credential is
...
+enhancement to {{?RFC9345}} has been standardized and deployed.
```
Comment (boucadair) [inline draft-rbw-add-encrypted-dns-forwarders.md]:
```suggestion
> enhancement to {{?RFC9345}} has been standardized and deployed.
```

**Review by boucadair — COMMENTED** [review]
the assumption identifies a gap. That should be tagged as such; hence the proposed note formatting. Feel free to adjust. Thanks

File: draft-rbw-add-encrypted-dns-forwarders.md (hunk 2 above)
Comment (tireddy2) [reply draft-rbw-add-encrypted-dns-forwarders.md]: we may want to say "enhancement to {{?RFC9345}} will be standardized and deployed."

**Review by boucadair — APPROVED** [review]

**Pull request merged**